test/fmf/tests/test-onboarding-tls.fmf

```diff
@@ -0,0 +1,24 @@
+summary: Complete FDO end-to-end protocol testing with TLS enabled
+description: |
+    Comprehensive FDO testing with TLS/HTTPS transport covering:
+    - TLS certificate generation for HTTPS transport
+    - Certificate infrastructure setup
+    - Multi-server orchestration with TLS (manufacturer, rendezvous, owner)
+    - Device initialization and onboarding over HTTPS
+    - ServiceInfo module testing
+    - Protocol validation with TLS
+
+test: ./test-onboarding.sh --tls
```

Comment:
I wouldn't use a flag for this, it is enough to set the `{manufacturing,owner,rendezvous}_http_protocol` variables to `https`